scripts/clusters.py

```diff
@@ -2509,4 +2509,16 @@
     # "ORFTEN30": {'snps': [29644], 'exclude_snps': [22226], 'cluster_data': [], "build_name": "ORF10.V30", "display_name": "ORF10:V30"}
     # "N220re": {'snps': [28931,29644], 'exclude_snps': [22226], 'cluster_data': [], "build_name": "N.A220Vre", "display_name": "N:A220Vre"}
     # "B1139" {'snps': [28486,28867], 'exclude_snps': [], 'cluster_data': [], "build_name": "B1139", "display_name": "B1139"}
+
+    "others": {
+        "build_name": "others",
+        "cluster_data": [],
+        "col": "#222222",
+        "snps": [],
+        "display_name": "others",
+        "graphing": False,
+        "important": False,
+        "nextstrain_build": False,
+        "type": "variant"
+    }

```

Comment:
This is the definition of the "others" variant in `clusters.py`. This object will be emitted into `clusters.json` alog with other variants. It should be the last object in the list.